### PR TITLE
Restructure Chialisp Site

### PR DESCRIPTION
**Pull request by Rigidity**
Content reorganization and rewrite, depends on similar changes made to the docs and dev guides

### CODE REVIEW

**Review by Starttoaster — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -4,27 +4,27 @@
...
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
     "serve": "docusaurus serve"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.1",
-    "@docusaurus/plugin-google-analytics": "^2.0.1",
-    "@docusaurus/plugin-google-gtag": "^2.0.1",
-    "@docusaurus/preset-classic": "2.0.1",
+    "@docusaurus/core": "2.0.0-beta.6",
```
Comment (pmaslana) [inline package.json:14]:
```suggestion
    "@docusaurus/core": "2.0.1",
```

Comment (pmaslana) [inline package.json:14]:
```suggestion
    "@docusaurus/core": "2.1.0",
```

File: package.json [hunk 2]
```diff
@@ -4,27 +4,27 @@
...
+    "@docusaurus/plugin-google-analytics": "^2.0.0-beta.14",
```
Comment (pmaslana) [inline package.json:15]:
```suggestion
    "@docusaurus/plugin-google-analytics": "^2.1.0",
```

File: package.json [hunk 3]
```diff
@@ -4,27 +4,27 @@
...
+    "@docusaurus/plugin-google-gtag": "^2.0.0-beta.14",
```
Comment (pmaslana) [inline package.json:16]:
```suggestion
    "@docusaurus/plugin-google-gtag": "^2.1.0",
```

File: package.json [hunk 4]
```diff
@@ -4,27 +4,27 @@
...
+    "@docusaurus/preset-classic": "2.0.0-beta.6",
```
Comment (pmaslana) [inline package.json:17]:
```suggestion
    "@docusaurus/preset-classic": "2.1.0",
```

File: package.json [hunk 5]
```diff
@@ -4,27 +4,27 @@
...
     "@easyops-cn/docusaurus-search-local": "^0.19.1",
     "@mdx-js/react": "^1.6.21",
-    "@svgr/webpack": "^6.0.0",
+    "@svgr/webpack": "^5.5.0",
```
Comment (pmaslana) [inline package.json:20]:
```suggestion
    "@svgr/webpack": "^6.0.0",
```

**Review by hoffmang9 — APPROVED** [review]
lgtm (f' it. ship it.)

**Review by Starttoaster — APPROVED** [review]
Re-approve bumped deps

**Pull request merged**